### ISSUE TITLE
If fine-tune parameters are consistent with the training parameters ?

### ISSUE THREAD

**Issue by yanjj16** [MyeongJin-Kim/Learning-Texture-Invariant-Representation#2]
I was wondering if your fine-tune parameters are consistent with the training parameters ? expecially during stage 1 ,when should I stop training the segmentation network?
Such as the following settings:
NUM_STEPS = 250000
POWER = 0.9
RANDOM_SEED = 1234
SAVE_NUM_IMAGES = 2
SAVE_PRED_EVERY = 5000
WEIGHT_DECAY = 0.0005

**Comment by MyeongJin-Kim** [comment]
Thanks for your interest!
I used same hyperparameters for stage1 and stage2.
Also I used stage1's best model to generate pseudo label and for fine-tuning.
